src/paperoni/__main__.py

```diff
@@ -288,6 +288,20 @@ def run(self, work: "Work"):
             work.collection.add_papers(selected)
             work.save()
 
+            work.focuses.update(work.collection.search(), config.autofocus)
+
+            focus_file = work.focus_file or (
+                yaml.safe_load(config._file).get("focuses", None)
+                if config._file
+                else None
+            )
+            if focus_file:
+                dump(Focuses, work.focuses, dest=focus_file)
+            # TODO: Not sure how to get the config file to update the focuses
+            # elif config._file:
+            #     _config: dict = yaml.safe_load(config._file)
+            #     _config["focuses"] = serialize(Focuses, work.focuses)
+
```

Comment:
I don't think this should be done in `paperoni work include`, but maybe a separate `paperoni focus update` command.